### PR TITLE
Fixed broken link to TSS

### PR DESCRIPTION
**Pull request by LanaStewa**
FR PR: https://github.com/canada-ca/systeme-conception/pull/253

### CODE REVIEW

File: continuous-improvement/choosing.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ title: Choosing what to improve
 
 <ul>
  <li>
-  <a href="./monitoring/GCTSS.html">
+  <a href="https://design.canada.ca/survey/index.html">
```
Comment (delisma) [inline continuous-improvement/choosing.md]: @prycrane  What's our URL strategy again? Are we doing relative, absolute or the root in a global variable?

Comment (prycrane) [reply continuous-improvement/choosing.md]: I don't think we ever formalized it, but root with a global variable is probably better for long-term maintenance? 

<a href="{{ site.url }}/survey/">

Comment (prycrane) [reply continuous-improvement/choosing.md]: `<a href="{{ site.url }}/survey/"> `

**Comment by LanaStewa** [comment]
Fixed the links as suggested and ready to go

**Review by LanaStewa — COMMENTED** [review]
All good to merge

**Comment by LanaStewa** [comment]
Also fixing broken image link: https://deploy-preview-259--systeme-conception-canada-ca.netlify.app/retroaction/etude-cas-ingredients

**Review by donohuech — APPROVED** [review]

**Pull request merged**